invenio_pidstore/cli.py

```diff
@@ -37,6 +40,28 @@
     from flask_cli import with_appcontext
 
 
+def loader_import(ctx, param, value):
+    """Import the loader."""
+    try:
+        return import_string(value)
+    except ImportError:
+        raise click.BadParameter('Loader {0} not found.'.format(value))
+
+
+def process_minter(ctx, param, value):
+    """Return the minter."""
+    with ctx.obj.load_app().app_context():
```

Comment:
Strange.. because in the docs they wrote: [Loads the Flask app (if not yet loaded) and returns it](https://github.com/inveniosoftware/flask-cli/blob/5cf611dd01788b92cfbbc6cf6cabed6ac1318a3d/flask_cli/cli.py#L209).

Maybe the `@cached_property` is not really useful because it called only one time. WDYT?